tests/unit_tests/utils/test_core.py

```diff
@@ -171,3 +174,30 @@ def test_other_values():
     assert cast_to_boolean([]) is False
     assert cast_to_boolean({}) is False
     assert cast_to_boolean(object()) is False
+
+
+def test_normalize_dttm_col() -> None:
+    """
+    Tests for the ``normalize_dttm_col`` function.
+
+    In particular, this covers a regression when Pandas was upgraded from 1.5.3 to
+    2.0.3 and the behavior of ``pd.to_datetime`` changed.
+    """
+    df = pd.DataFrame({"__time": ["2017-07-01T00:00:00.000Z"]})
+    assert (
```

Comment:
@betodealmeida why is this check needed? As expected there's nothing up Pandas's sleeve when you create the DataFrame.

Reply:
It's not needed, it's just to show what's in the dataframe if anyone's reading at this test. I'm happy to remove it.